apps/ewallet/lib/ewallet/web/paginator.ex

```diff
@@ -67,7 +67,20 @@ defmodule EWallet.Web.Paginator do
   # Returns the per_page number or default, but never greater than the system's defined limit
   defp get_per_page(attrs) do
     per_page = Map.get(attrs, "per_page", @default_per_page)
-    max_per_page = Application.get_env(:ewallet, :max_per_page, @default_max_per_page)
+
+    max_per_page =
+      case Application.get_env(:ewallet, :max_per_page, @default_max_per_page) do
```

Comment:
How come this one we need to handle the `{:system, _}` tuple ourselves while other configs don't need to?

Reply:
Because `ewallet` the module isn't an app so we don't have exact timing to call `DeferredConfig`.